english/marshmallow.md

```diff
@@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
 
 Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
 
-Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
```

Comment:
Sorry, I didn’t intentionally delete any of those lines.  I’ll see what happened and resubmit.

> On Oct 30, 2015, at 4:25 PM, Casey Collaborator <notifications@github.com> wrote:
> 
> In english/marshmallow.md <https://github.com/udacity/create-your-own-adventure/pull/2685#discussion_r43562342>:
> 
> > @@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
> >  
> >  Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
> >  
> > -Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
> Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging. If you wish to have this pull request auto-merged, please update it to only add lines. If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/udacity/create-your-own-adventure/pull/2685/files#r43562342>.
>